lib/import.js

```diff
@@ -124,7 +125,8 @@ class Import {
 
                 // validate the geojson file before uploading starts
                 const geojsonErrs = validateGeojson(filepath, {
-                    ignoreRHR: options.ignoreRHR
+                    ignoreRHR: options.ignoreRHR,
+                    schema: getSchema()
```

Comment:
You'll need to move this above geojson Errs. Right now this won't return synchronously and will result in the schema being null (or a function) every time.